chain-signatures/node/src/protocol/mod.rs

```diff
@@ -304,7 +315,7 @@ impl MpcSignProtocol {
                 let from_state = format!("{state}");
                 state = match state.advance(&mut self, contract_state).await {
                     Ok(state) => {
-                        tracing::debug!("advance ok: {from_state} => {state}");
+                        tracing::trace!("advance ok: {from_state} => {state}");
```

Comment:
not sure if @ppca will like this because it might not be visible that we're advancing states in our normal debug logs

Reply:
I've made it trace because it is too spammy. We should fix it later with logging spans.